examples/testimonials/readme.md

```diff
@@ -0,0 +1,15 @@
+# Forms example - Testimonial
+This is an example of using a form submission to automatically create a piece type in your Apostrophe 3 project.
+## Installation
+* Copy the `modules/@apostrophe/form` folder into your project
+* Copy the `modules/testimonial` folder into your project
+* Add the testimonial module to app.js as `testimonial: {},`
+## Extending the form module
+* `form/index.js` adds the async function `createTestimonial`, which is called on form submission.  This calls the `testimonial.addTestimony()` function to create our new testimonial piece.
+## The testimonial piece
+### `index.js`
+The fields `names`, `emailaddress`, `phonenumber`, `testimonial` and `helpneeded` need to be matched by fields in the created form.  When the form is submitted, that data is passed to `addTestimony`, these fields build our new piece.  There are two extra fields in the `testimonial` piece type:
```

Comment:
I would say something like: "We will automatically fill five `string` type fields in the testimony piece type. Each of these fields need to be in the form you present to the user. They are .... You should choose the input types to match the expected input in your form. When the form is submitted, ..."